pkgs/servers/mail/listmonk/frontend.nix

```diff
@@ -0,0 +1,29 @@
+{ yarn2nix-moretea
+, fetchFromGitHub
+}:
+
+yarn2nix-moretea.mkYarnPackage rec {
+  pname = "listmonk-frontend";
+
+  packageJSON = ./package.json;
+  yarnLock = ./yarn.lock;
+  yarnNix = ./yarn.nix;
+
+  version = import ./version.nix;
+
+  installPhase = ''
+    cd deps/listmonk-frontend/frontend
+    npm run build
+
+    mv dist $out
+  '';
+
+  doDist = false;
+
+  src = fetchFromGitHub {
+    owner = "knadh";
+    repo = "listmonk";
+    rev = "v${version}";
+    sha256 = import ./source-hash.nix;
+  };
+}
```

Comment:
meta is missing. We can probably inherit it from the backend.

Reply:
Done.